crates/pathfinder/src/sequencer/error.rs

```diff
@@ -37,7 +37,7 @@ impl From<SequencerError> for rpc::Error {
                 StarknetErrorCode::BlockNotFound if e.message.contains("Block number") => {
                     RpcErrorCode::InvalidBlockNumber.into()
                 }
-                _ => rpc::Error::Call(rpc::CallError::Failed(e.into())),
+                _ => Error::Call(CallError::Failed(e.into())),
```

Comment:
Could you leave off switching uses around and leaving the PR with changes required by the upgrade only? It will be easier to review that way.

Reply:
In this case jsonrpsee now exports these types off different modules now, so they can't share the `uses`. i can fully qualify them though

Reply:
Aah I see! Apologies then, this style seems less verbose and all good.